recipes/ffmpeg/all/conanfile.py

```diff
@@ -179,10 +185,6 @@ class FFMpegConan(ConanFile):
     generators = "pkg_config"
```

Comment:
Is it really required??

Reply:
If you remove it you get the next error:
```
ERROR: ffmpeg/5.0@lumicks/cookbook-testing: Error in build() method, line 569
	shutil.copy("x264.pc", "libx264.pc")
	FileNotFoundError: [Errno 2] No such file or directory: 'x264.pc'
```